recipes/gnuradio-symbolmapping/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set oot_name = "symbolmapping" %}
+{% set name = "gnuradio-" + oot_name %}
```

Please avoid introducing more Jinja variables than necessary
```suggestion
{% set name = "gnuradio-symbolmapping" %}
```